Assets/Scripts/Nexus.cs

```diff
@@ -5,10 +5,22 @@
 
 public class Nexus : Tower
 {
-    public int pPidOwner;
-    public Image healthBar;
+  public int pPidOwner;
+
+  protected override void Start()
+  {
+    MaxHealth = 30;
+    health = 10; //FIXME

```

Comment:
Pourquoi pas les avoir un public au dessus pour regler depuis l editor

Reply:
It was to make test quicker but i removed it